packages/plugin-ext/src/main/browser/plugin-ext-frontend-module.ts

```diff
@@ -139,4 +140,6 @@ export default new ContainerModule((bind, unbind, isBound, rebind) => {
     rebind(DebugService).toService(PluginDebugService);
     bind(PluginDebugSessionContributionRegistry).toSelf().inSingletonScope();
     rebind(DebugSessionContributionRegistry).toService(PluginDebugSessionContributionRegistry);
+
+    bindFileSystemPreferences(bind);
```

Comment:
it is not required, https://github.com/theia-ide/theia/pull/4721/files#diff-748e5ff8fc3869ccdac5ee2cc473481e is enough